Doubly_Linked_List/Doubly_Linked_List.java

```diff
@@ -0,0 +1,238 @@
+package LinkedList;
+
+import java.util.*;
+
+class Doubly_Linked_List {
+
```

Comment:
remove this line too

Reply:
@asha15 Mam without these lines the code won't work.